lib/litmus_paper/metric/unix_socket_utilization.rb

```diff
@@ -27,6 +27,15 @@ def current_health
         end
       end
 
+      def stats
+        current_health
+        {
+          :socket_active => @active,
+          :socket_queued => @queued,
+          :socket_utilization => ((@queued / @maxconn.to_f) * 100).round,
```

Comment:
This method is returning the data from the instance variables of this class, that data is only as fresh as the last time this particular unicorn process was health checked, is that the behavior you want? Instead shouldn't the health checks persist the stats information to a file and then the stats methods return the latest data from the file?

Reply:
We actually refresh these instance variables immediately before they're used by invoking `current_health` on line 31. Given the rate at which we expect this endpoint to be hit we're also not concerned about overloading the system refreshing these metrics on each call.

I think the data-freshness levels as we have them are OK

Reply:
I've removed the instance var setting to make it a bit more clear when values are being fetched